src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -273,6 +273,12 @@ public void deleteTicket(long id) {
         complete(submit(req("DELETE", tmpl("/tickets/{id}.json").set("id", id)), handleStatus()));
     }
 
+    public ListenableFuture<JobStatus<Ticket>> createTicketAsync(Ticket ticket) {
+        return submit(req("POST", cnst("/tickets.json?async=true"),
```

Comment:
Is it possible to create a ticket without `async=true` but with a `ListenableFuture` as a return type?

Reply:
If you can come up with a good name for it so we can have both i'll put it right in ;)

Reply:
`ListenableFuture<JobStatus<Ticket>> createTicketAsync(Ticket ticket, boolean sendToBackground)` ? :)

Reply:
one returns a job status, the other returns an actual ticket https://developer.zendesk.com/rest_api/docs/core/tickets#create-ticket

Reply:
`ListenableFuture<Ticket> createTicketAsync(Ticket ticket)`
`ListenableFuture<JobStatus<Ticket>> createTicketDeferred(Ticket ticket)`

? :)

Reply:
I think that works I'll update the pr when I get back to the keyboard.